contracts/HydraStaking/modules/PenalizeableStaking/PenalizeableStaking.sol

```diff
@@ -55,6 +47,9 @@ contract PenalizeableStaking is IPenalizeableStaking, HydraChainConnector, Staki
         }
 
         uint256 leftToWithdraw = leftToWithdrawPerStaker[msg.sender];
+        if (leftToWithdraw == 0) {
+            revert NoFundsToWithdraw();
+        }
         delete leftToWithdrawPerStaker[msg.sender];
```

Comment:
@SamBorisov, do you think that it will be nice to have some space between part of the lines here? Maybe between the delete line and the next two calls.